core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -253,13 +257,15 @@ object ConsoleConsumer extends Logging {
       .withRequiredArg
       .describedAs("deserializer for values")
       .ofType(classOf[String])
+    val enableLifecycleLoggingOpt = parser.accepts("enable-lifecycle-logging", "Log lifecycle events of the consumer.")
```

Comment:
I think it's better to extend to more events later when we use them.. VerifiableConsumer has more of these events. But it also seems ok to add a flag for future additions of events when needed.

I think you can use both 'print' or 'log' as a more general term, no?  logging to standard output